containenv/commands/containenv.py

```diff
@@ -219,3 +234,41 @@ def write_runcontainenv(self):
     def publish_image(self):
         '''if there's a shared registry publish to it'''
         pass
+
+
+class Activate(object):
+    '''Usage: containenv activate [help]
```

Comment:
What does `containenv activate help` do? The built in subcommand help is `containenv help activate`. Custom help message?